src/routes/ProfilePage/ProfilePage.js

```diff
@@ -96,11 +108,53 @@ export default class ProfilePage extends Component {
   }
 
   renderPreferences() {
-    // TODO: add after userPreferences backend is complete  
+    const prefs = this.state.userPreferences;
+    let animalTypes = prefs.TypeOfAnimal ? JSON.parse(prefs.TypeOfAnimal) : null;
+    let breeds = prefs.Breed ? JSON.parse(prefs.Breed) : null;
+    let more = prefs.More ? JSON.parse(prefs.More) : null;
+
+    // Format age range
+    let ageRange;
+    if (prefs.MinAge && prefs.MaxAge) {
+      ageRange = prefs.MinAge + '-' + prefs.MaxAge;
+    } else if (prefs.MinAge) {
+      ageRange = prefs.MinAge + '+';
+    } else if (prefs.MaxAge) {
+      ageRange = '< ' + prefs.MaxAge;
+    }
+
+    // Format objects
+    if (animalTypes) {
+      animalTypes = animalTypes.map((elem) => {
+        return elem.value;
+      }).join(', ');
+    };
+    if (breeds) {
+      breeds = breeds.map((elem) => {
+        return elem.value;
+      }).join(', ');
+    };
+    if (more) {
+      more = more.map((elem) => {
+        // This will add a space before any capital letter, and then lowercase
+        // Simply for formatting purposes
+        return elem.value.replace(/([a-z])([A-Z])/g, '$1 $2').toLowerCase();
+      }).join(', ');
+    };
+
     return (
       <Section>
         <>
-          <h2>Saved Preferences</h2>
+          <h2>Saved Preferences <span className='btn p-2 text-primary' onClick={this.deleteAllPreferences}><FontAwesomeIcon icon='trash-alt' /></span></h2>
+          <table className='table'>
+            <tbody>
+              {animalTypes ? <tr><th>Types of Animals</th><td>{animalTypes}</td></tr> : null}
```

Comment:
I am being picky but what do you think about changing the text alignment to left? 
I think left alignment looks better?
![image](https://user-images.githubusercontent.com/71732016/156064965-c5b1767c-ba45-48b2-87e9-286c452907f0.png)

![image](https://user-images.githubusercontent.com/71732016/156064979-7a867830-35a7-4c6f-a161-0ef612ff7c22.png)

Reply:
How did you make the text closer together? I tried `className='align-left'` and everything stayed where it was